model_package_core/tests/test_dataset.py

```diff
@@ -0,0 +1,28 @@
+from model_package_core.constants import (TRAIN_IMAGE_DATA_PATH,
+                                          TRAIN_LABEL_DATA_PATH)
+from model_package_core.pytorch_model.dataset import ProcessedDataset
+
+
+def test_processed_dataset_class():
+    # GIVEN a path to raw training data images and labels
+    # and a specified number of training samples
+    from pathlib import PosixPath
+    assert type(TRAIN_IMAGE_DATA_PATH) == PosixPath
+    assert type(TRAIN_LABEL_DATA_PATH) == PosixPath
+    number_of_samples = 320
+
+    # WHEN a new ProcessedDataset is created
+    dataset = ProcessedDataset(
+        TRAIN_IMAGE_DATA_PATH,
+        TRAIN_LABEL_DATA_PATH,
+        number_of_samples
+    )
```

Comment:
As my comment in the constants.py alluded to, the tests fail if the computer running the tests doesn't have these very specific training data files. If tests depend on some external data, the data must be committed to the repo. This is where fixtures shine. I'd recommend curating an extremely small set of training data and committing it to a fixtures dir within the test directory (ie `model_package_core/tests/fixtures/train_image` `model_package_core/tests/fixtures/train_label`), then writing a pytest fixture to return the path to those fixture files.